cypress/integration/RichTextInput/Rich_Text_Keyboard_Shortcuts.feature

```diff
@@ -0,0 +1,34 @@
+Feature: Keyboard shortcuts inside an input when rich text is enabled
+
+  Scenario Outline: The user uses a keyboard shortcut with text selected
+    Given there is a text string selected
+    When the user presses <shortcut>
+    Then the text string is formatted <result>
+    And the text string is still selected
+
+    Examples:
+      | shortcut | result |
+      | ctrl+b   | bold   |
+      | cmd+b    | bold   |
+      | ctrl+i   | italic |
+      | cmd+i    | italic |
+
+  Scenario Outline: The user uses a keyboard shortcut without text selected
+    Given there is focus inside the rich text enabled input
+    When the user presses <shortcut>
+    Then the <format> tags are added
+    And the text cursor is in the middle of the added tags
```

Comment:
This one is a bit unclear. What are the tags added to? Around the word under the cursor? The whole line? (Like when selecting "Headline", "Paragraph", etc.) What happens if the cursor is on a space character?

I assume this is meant to change the word under the cursor, so I think we should specify that

```feature
And the cursor is on a word
# ...
Then the <format> tags surround the word under the cursor
```

Reply:
Good point. You're right, if the cursor is inside a word then that word will get the formatting. I've adjusted that one scenario to account for this and added a new scenario for the `inside a word` example. I've also added this to the toolbar file too in: `The user clicks a formatting button with the cursor inside a word`.